README.ksonnet.md

```diff
@@ -0,0 +1,213 @@
+# Using Ksonnet 
+
+Kubeflow is currently evaluating different strategies for allowing deployments to be customized ([google/kubeflow/issues/23](https://github.com/google/kubeflow/issues/23)).
+
+This directory contains a prototype for using ksonnet to help us evaluate ksonnet.
+
+If you are unfamiliar with ksonnet you may want to start by reading the [tutorial](https://ksonnet.io/docs/tutorial)
+
+## Build ks
+
+You need a version of ksonnet newer than the [0.7.0 release](https://github.com/ksonnet/ksonnet/releases).
+
+As of this writing, Heptio hasn't released any newer prebuilt binaries so you will need to [build from source](https://ksonnet.io/docs/build-install).
+
+```
+# Clone the ksonnet repo into your GOPATH
+go get github.com/ksonnet/ksonnet
+cd ${GOPATH}/src/github.com/ksonnet/ksonnet
+make install
+```
+
+## Deploy Kubeflow
+
+Initialize a directory to contain your deployment
+```
+ks init my-kubeflow
+```
+
+Install the Kubeflow packages
+
+```
+cd my-kubeflow
+ks registry add kubeflow github.com/google/kubeflow/tree/master/kubeflow
+ks pkg install kubeflow/core
+ks pkg install kubeflow/tf-serving
+ks pkg install kubeflow/tf-job
+```
+
+
+Create the Kubeflow core component. The core component includes 
+  * JupyterHub
+  * TensorFlow job controller
```

Comment:
Rename `core` to `training`?

Reply:
Naming something `core` indicates that it's a dependency for other subsequent steps, which is not strictly true.

Reply:
JupyterHub isn't training specific and the CRD is needed for subsequent steps. The idea here is that we have certain core components that you probably only deploy once. Whereas for tf-serving and tf-job you probably have multiple ones.

I'm open to other names but training doesn't seem right.